Java-Basic-Inheritance.md

```diff
@@ -0,0 +1,51 @@
+# Java Inheritance
+
+So great you have successfully created a Car class. But, wait, aren't Tesla cars supposed to be electric variants? I want an Electric car class, but it also should have the properties of the original `Car` class.
```

Comment:
> So great you have successfully created a Car class.

Maybe phrase it differently, with proper reference to where they had created the `Car` class?